packages/jellyfish-api-core/__tests__/category/blockchain/getBlockchainInfo.test.ts

```diff
@@ -0,0 +1,82 @@
+import { MasterNodeRegTestContainer, RegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import waitForExpect from 'wait-for-expect'
```

Comment:
For this kind of these where you spin up 2 nodes you should do it like this instead:

You could just call it, it's cleaner without nesting it again.

```ts
describe('BlockchainInfo without masternode', () => {
})

describe('BlockchainInfo on masternode', () => {
})
```

```ts
describe('BlockchainInfo', () => {
  describe('without masternode', () => {
  })

  describe('on masternode', () => {
  })
})
```